src/rvs_blas.cpp

```diff
@@ -437,8 +436,8 @@ bool rvs_blas::run_blass_gemm(std::string ops_type) {
                   int sol_index = 0;
                   int flags = 10;
 
-                  alpha = blas_alpha_val;
-                  beta  = blas_beta_val;
+                  alpha.data = blas_alpha_val;
+                  beta.data  = blas_beta_val;
```

Comment:
This is wrong data conversion.  If you call hgemm you need to use a IEEE float16 representation.  You may need compiler flags and then just _Float16 alpha = (float)blas_alpha_value.     I add the cast as there was a bug in double to _Float16 in recent versions of the compiler.   Rocblas has a define ROCM_USE_FLOAT16 for internally not using the struct.  .data is an unsigned short so you would have serious data loss.

Reply:
will fix this